src/routes/login/+page.svelte

```diff
@@ -1 +1,98 @@
-<h1>Login</h1>
+<script lang="ts">
+	let email = '';
+	let password = '';
+	let isLoading = false;
+	let isSuccess = false;
+
+	let errors = {
+		email: undefined,
+		password: undefined,
+	};
```

Comment:
These seem to be unused. Let's remove them for now until we need them to avoid confusion?